exercises/diamond/examples/success-standard/src/Diamond.hs

```diff
@@ -0,0 +1,13 @@
+module Diamond (diamond) where
+
+import Data.Char (ord, chr)
+
+diamondH :: (Int -> String) -> (Char, (Int, Int)) -> String
+diamondH rb (c, (0, y)) = rb y ++ [c] ++ rb y
```

Comment:
I am having a hard time understanding what the H in diamondH stands for. I would prefer a name that I didn't have to ask someone to explain. This makes a single row, right? Maybe something about that.